<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEndPage</class>
 <widget class="QWidget" name="CEndPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>72</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>游戏结束！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBNewGame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>新游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBExitGame">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelScore">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>您的分数：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
